### PR TITLE
Twolame: bump

### PR DESCRIPTION
**Pull request by extrowerk**
Remove unneded patch
Add _bin subpackage

### CODE REVIEW

File: media-sound/twolame/twolame-0.4.0.recipe [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+on tooLAME by Mike Cheng, which in turn is based upon the ISO dist10 code and \
+portions of LAME."
+HOMEPAGE="https://www.twolame.org"
+COPYRIGHT="2004-2018 The TwoLAME Project"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/twolame/twolame-$portVersion.tar.gz"
+CHECKSUM_SHA256="cc35424f6019a88c6f52570b63e1baf50f62963a3eac52a03a800bb070d7c87d"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```
Comment (waddlesplash) [inline media-sound/twolame/twolame-0.4.0.recipe:12]: Not `all`?

**Comment by extrowerk** [comment]
Done

File: media-sound/twolame/twolame-0.4.0.recipe [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+PROVIDES="
+	twolame$secondaryArchSuffix = $portVersion compat >= 0
+	lib:libtwolame$secondaryArchSuffix = 0.0.0 compat >= 0
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libFLAC$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libsndfile$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
```
Comment (Begasus) [inline media-sound/twolame/twolame-0.4.0.recipe:24]: Looking at configure checks, libFLAC, libogg and libvorbis are not used (neither in REQUIRES nor REQUIRES_devel), (no mention from pkg-config also), ports like Only libsndfile should be needed (checked some recipes at repology for this)

File: media-sound/twolame/twolame-0.4.0.recipe [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	libtoolize --force --copy --install
+	aclocal
+	autoconf
+	autoheader
+	automake --add-missing
```
Comment (Begasus) [inline media-sound/twolame/twolame-0.4.0.recipe:70]: Maybe combine these with `autoreconf -vfi`?

**Review by Begasus — COMMENTED** [review]
Some suggestions, there is also a test case that you could enable with `make check`, maybe remove the older recipe too?

**Pull request merged**